homeassistant/components/automation/strings.json

```
@@ -25,7 +25,7 @@
           "state": {
             "parallel": "Parallel",
             "queued": "Queued",
-            "restart": "Restart",
+            "restart": "[%key:common::action::restart%]",
```

This is the only one that I have doubts over. 

Easy to revert if we want.
```suggestion
            "restart": "Restart",
```

---

Yeah, you mean it could be mixed up with a system restart ?
But isnt't that called the same ? You restart either an automation or HA itself ?

---

I think it should not be set to the common word as it's a mode and not an action and therefore it probably is translated differently.
I would probably translate it differently in Swedish at least.

---

ah, that makes sense. So it should probably get its own key like mode_restart ?

---

To me this sounds like a strange state, eg "What is this automation doing?" "Restart". (Parallel is also strange here, queued seems a better fit). I'm not an expert on the automation internals, but wouldn't "Restarting" make more sense?

---

> To me this sounds like a strange state, eg "What is this automation doing?" "Restart". (Parallel is also strange here, queued seems a better fit). I'm not an expert on the automation internals, but wouldn't "Restarting" make more sense?

It's not a state but a default action for the automation and queued is actually another possibility. Restart is the right context, now just the wording .. Maybe just "abort and start over" ?